tests/openapi/test_strictmode.py

```diff
@@ -1631,100 +1637,76 @@ def scroll_request(condition_count: int):
     assert response.status_code == 429
     assert "Read rate limit exceeded: Operation requires 7 tokens" in response.json()['status']['error']
 
+def test_read_rate_limiter_many_vectors(full_collection_name):
+    collection_name = full_collection_name
 
-def test_strict_mode_group_limits(collection_name):
```

Comment:
Did you just nuke my tests? :smile:

Reply:
Sorry, looks like I mishandled conflicts when rebasing

Reply:
https://github.com/qdrant/qdrant/pull/6356/commits/d2cf3856673c1bce700128b31b96efa74f3b6f50